projects/asthma-copd/nbs/30_gls_phenoplier/01-run_gls_phenoplier.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        GENE_CORR_FILE="${PHENOPLIER_PROJECTS_ASTHMA_COPD_RESULTS_DIR}/gls_phenoplier/gene_corrs/${pheno_id}/gene_corrs-symbols.per_lv"
</pre><p>Line is a but long, not sure if it's possible to split it though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Yeah, I remember trying to split it before. Unfortunately, it doesn't seem to be very easy in bash.</p>